e2e/sync_test.go

```diff
@@ -1793,3 +1795,43 @@ func TestIssue435(t *testing.T) {
 		assertError(t, err, errS3NoSuchKey)
 	}
 }
+
+// If source path contains a special file it should not be synced
+func TestSyncSocketDestinationEmpty(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	workdir := fs.NewDir(t, t.Name())
+	defer workdir.Remove()
+
+	sockaddr := workdir.Path() + "/sock"
```

Comment:
`filepath.Join` should be the preferred way of creating local file system paths.

Reply:
Resolved. https://github.com/peak/s5cmd/pull/618/commits/a0f01010ed4e222e9565caec46124517861ad357